src/main/java/org/jenkinsci/gradle/plugins/jpi2/PrepareServerTask.java

```diff
@@ -0,0 +1,44 @@
+package org.jenkinsci.gradle.plugins.jpi2;
+
+import org.apache.commons.io.FileUtils;
+import org.gradle.api.DefaultTask;
+import org.gradle.api.artifacts.Configuration;
+import org.gradle.api.tasks.TaskAction;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+
+public class PrepareServerTask extends DefaultTask {
+    private Configuration serverPluginsClasspath;
+    private File jpi;
+    private String projectRoot;
+
+    public void setServerPluginsClasspath(Configuration serverPluginsClasspath) {
```

Comment:
We don't want these to be considered task inputs?

Reply:
I think we can do that. I'll create a follow-up PR. I'm guessing I can replace this with a `Copy` task now that it only prepares the server.